charts/logs-dashboard.json

```diff
@@ -1165,6 +1165,11 @@
         "label": null,
         "multi": true,
         "name": "level",
+        "current": {
+          "selected": true,
+          "text": "All",
+          "value": "$__all"
+         },
```

Comment:
```suggestion
        },
```

Reply:
Done ! Yes I tested it on my side and it fix the issue.